### PR TITLE
Retiring AT2 and Garfield

### PR DESCRIPTION
**Pull request by ineiti**
- Adds images to the demos of garfield and at2.
- cleaned up display of incubated and retired projects
- made more consistent description of the `incubator:work:` descriptions

### CODE REVIEW

File: views/products/demo/at2.tpl [hunk 1]
```diff
@@ -6,18 +6,59 @@
...
+    </div>
+</div>
+<div class="columns">
+    <div class="click-zoom">
+        <label>
+            <input type="checkbox">
+            <img src="../../../resources/products/images/at2/at2-demo-3.png">
+            <br>Now you can transfer some assets to one of the other accounts
+            on the network. For this test you have 100k coins of whatever value
+            (currently 0US$ :)
```
Comment (PascalinDe) [inline views/products/demo/at2.tpl:35]:
```suggestion
            (currently 0US$ :))
```

**Comment by PascalinDe** [comment]
general comment is that for images adding alt text is recommended to improve accessibility

File: views/products/demo/at2.tpl (hunk 1 above)
Comment (ineiti) [reply views/products/demo/at2.tpl:35]: Personal preference: I'm used to close the parentheses with a smiley...

File: data/DEDIS/projects.yaml [hunk 2]
```diff
@@ -115,7 +115,7 @@ projects:
     incubator:
       type: retired_archived
       work: |
-        The C4DT created a demonstrator for Calypso, as well as a hands-on workshop.
+        2019/01 - The C4DT created a demonstrator for Calypso, as well as a hands-on workshop - retired in 2022/10
```
Comment (PascalinDe) [inline data/DEDIS/projects.yaml:118]:
```suggestion
        2019/01 - The C4DT created a demonstrator for Calypso, as well as a hands-on workshop - retired in 2024/10
```

**Review by PascalinDe — APPROVED** [review]
LGTM - great alt texts! :)

**Pull request merged**